packages/mc-dev-authentication/html-docs/login.html

```diff
@@ -0,0 +1,228 @@
+      </h3>
+      <small>
+        This page is only available in development mode and is necessary to
+        authenticate yourself. In production environment, we use our own
+        authentication service.
+      </small>
+    </div>
+    <form id="login">
+      <div id="errors"></div>
+      <div class="field">
+        <label for="email">
+          Email<abbr title="This field is mandatory">*</abbr>
+        </label>
+        <input id="email" name="email" type="text" required="required" />
+      </div>
+      <div class="field">
+        <label for="password">
+          Password<abbr title="This field is mandatory">*</abbr>
+        </label>
+        <input
+          id="password"
+          name="password"
+          type="password"
+          required="required"
+        />
+      </div>
+      <div>
+        <button type="submit" aria-label="Sign in">Sign in 🚀</button>
+      </div>
+      <div class="info">
+        <small>
+          Note that Single Sign On is not supported at the moment for the
+          development login page. If you are interested in this functionality,
+          let us know and open a
+          <a
+            href="https://github.com/commercetools/merchant-center-application-kit/issues/new/choose"
+            target="_blank"
+            rel="noopener noreferrer"
+            >support issue</a
+          >.
+        </small>
+      </div>
+    </form>
+    <script>
+      /* eslint-disable no-var,vars-on-top */
+      /**
+       * NOTE:
+       *   This code is only used in development mode.
+       *   It authenticates a developer using the same mechanisms
+       *   as when not running in development. However,
+       *   this runs on the same domain as the developer.
+       */
+      window.addEventListener('load', function loaded() {
+        var form = document.getElementById('login');
+        form.addEventListener('submit', function onSubmit(event) {
+          event.preventDefault();
+          authorize();
+        });
+
+        function authorize() {
+          var data = new FormData(form);
+          var payload = {
+            email: data.get('email'),
+            password: data.get('password'),
+          };
+
+          var queryParams = new URLSearchParams(window.location.search);
+          if (queryParams.has('response_type')) {
+            // OIDC params
+            payload.client_id = queryParams.get('client_id');
+            payload.response_type = queryParams.get('response_type');
+            payload.scope = queryParams.get('scope');
+            payload.state = queryParams.get('state');
+            payload.nonce = queryParams.get('nonce');
+          }
+
+          var container = document.getElementById('errors');
+          // Clean up error message elements
+          while (container.firstChild) {
+            container.removeChild(container.firstChild);
+          }
+
+          const url = '__MC_API_URL__/tokens';
```

Comment:
This is the only part that "changed". We now simply do a string replacement for this variable.